### DISCUSSION TITLE
Is there an option to output multiple formats, but not all?

### DISCUSSION THREAD

**Discussion by xtreamrob24** [Purfview/whisper-standalone-win#95] [Ideas]
So I found out --skip only recognises .srt files.
As an easy workaround I though I might just output only plain text (my desired format) and .srt (for the --skip option to work). 
Is there any way to do that? I am pretty new to using cmd and have only very basic programming skills - which is also why I am forever grateful for the whisper-standalone!

(One option would be of course to just output all formats with --output_format=all, then group the files by filetype and delete whatever I dont need. But is there something more elegant?)

**Comment by Purfview** [comment]
Currently for output you can select only one format or all.

**Reply by xtreamrob24** [reply]
Okay, thanks :)

**Comment by Purfview** [comment]
There I found such implementation -> https://github.com/openai/whisper/pull/1039

That code for me doesn't look clean, if someone makes it right then I would consider adding it, I personally don't want to bother with it.

**Comment by Purfview** [comment]
Implemented since r194.1
